lib/rainforest_cli/csv_importer.rb

```diff
@@ -27,9 +27,19 @@ def import
       end
       raise 'Invalid schema in CSV. You must include headers in first row.' if !columns
 
-      print 'Creating custom step variable'
+      puts 'Checking for existing tabular variables.'
+      generators = http_client.get('/generators')
+      generator = generators.find { |g| g['name'] == @generator_name }
+
+      if generator
```

Comment:
Can you add a new CLI option (`--overwrite` or something)? Don't want anyone to trigger this by accident.